bug_monitor_add.php

```diff
@@ -55,26 +55,34 @@
 
 $f_bug_id = gpc_get_int( 'bug_id' );
 $t_bug = bug_get( $f_bug_id, true );
-$f_username = gpc_get_string( 'username', '' );
+$f_usernames = trim( gpc_get_string( 'username', '' ) );
 
 $t_logged_in_user_id = auth_get_current_user_id();
 
-if( is_blank( $f_username ) ) {
-	$t_user_id = $t_logged_in_user_id;
+if( is_blank( $f_usernames ) ) {
+	$t_user_ids = array( $t_logged_in_user_id );
 } else {
-	$t_user_id = user_get_id_by_name( $f_username );
-	if( $t_user_id === false ) {
-		$t_user_id = user_get_id_by_realname( $f_username );
-
-		if( $t_user_id === false ) {
-			error_parameters( $f_username );
-			trigger_error( ERROR_USER_BY_NAME_NOT_FOUND, E_USER_ERROR );
+	$t_usernames = array( $f_usernames );
+	foreach( array( "|", "," ) as $t_sep ) {
+		if( strpos( $f_usernames, $t_sep ) !== false ) {
+			$t_usernames = explode( $t_sep, $f_usernames );
+			break;
```

Comment:
Wouldn't it be simpler to do 
``` php
$t_usernames = preg_split( '/[,|]/', $f_usernames, -1, PREG_SPLIT_NO_EMPTY )
```
And maybe add an `array_unique()` call too, just in case.